tests/test_util.py

```diff
@@ -146,72 +133,6 @@ def noop():
         process.start.assert_called_with()
 
 
-def test_create_tx_with_empty_inputs():
```

Comment:
Actually, I think this one is not in there: https://github.com/bigchaindb/bigchaindb/pull/641/files#diff-5832adf374920d75d4bf48e546367f53L165

I'll create a ticket on BDB-common